migrations/schema.sql

```diff
@@ -130,6 +130,7 @@ CREATE TABLE IF NOT EXISTS `pulls` (
   `state` enum('open','closed') COLLATE utf8mb4_general_ci NOT NULL,
   `title` varchar(255) COLLATE utf8mb4_general_ci NOT NULL,
   `body` text COLLATE utf8mb4_general_ci NOT NULL,
+  `draft` boolean default false,
```

Comment:
We also need a migration file to make this mutation. Use the `AFTER {column}` syntax too.

Reply:
Added migration file via https://github.com/iFixit/pulldasher/pull/329/commits/b6cbe56787129ad52255d4b7cac2fac5f9c5b8dd